articles.tex

```diff
@@ -680,4 +680,39 @@
 The Judicial Board may present an official ruling following the Judicial Investigation.
 This ruling may be appealed in the same manner as an Executive Board decision \ref{Appeals}.
 
+% ARTICLE VII - TEMPORARY AMENDMENTS
+\article{Temporary Amendments}
+In the case that an amendment should be temporary, the amendment's changes should be listed here. 
+Each of these amendments is given a designated period of time, and will cease to exist immediately following the end of the period.
```

Comment:
Replace ```is``` with 'are'.  "```Each of these amendments``` are ```given a...```"

Reply:
I'll change it once the amendment passes. That way everyone can vote yes to the same wording. Its a non-semantic change but ya know.

Reply:
I believe it's actually correct as-is. After saying "each" you're referring to the singular item not the plural.